src/main/java/net/dv8tion/jda/api/requests/restaction/ChannelAction.java

```diff
@@ -49,6 +50,14 @@
     @Override
     ChannelAction<T> setCheck(@Nullable BooleanSupplier checks);
 
+    @Nonnull
+    @Override
+    ChannelAction<T> timeout(long timeout, @Nonnull TimeUnit unit);
+
+    @Nonnull
+    @Override
+    ChannelAction<T> deadline(long timestamp);
```

Comment:
Should this `@inheritDoc` ?

Reply:
No, that is unnecessary. I'm not sure why its present in AuditableRestAction, might just be a mistake on my end from back when i wrote it.